src/app/body/task-details/task-details.component.css

```diff
@@ -34,7 +34,7 @@
 }
 
 .sprint {
-    color: #5A20F0;
+    color: #10BE65;
```

Comment:
Is this change needed?

Reply:
Yes, you had asked me to change the task ID color, so this is the style for that

Reply:
This is the green color used throughout the app